Unable to display Help panel or simply get version when `ecs.php`  config file is invalid

Recently I've trying to implement a Git pre-commit Hook with [CaptainHook](https://github.com/captainhook-git/captainhook) running ECS.

On my first run, as I've no `ecs.php` config file ECS created a default file

```text 
 No "ecs.php" config found. Should we generate it for you? [yes]:
 >


 [OK] The ecs.php config was generated! Re-run the command to tidy your code

```

with such contents


```php
<?php

declare(strict_types=1);

use PhpCsFixer\Fixer\Import\NoUnusedImportsFixer;
use Symplify\EasyCodingStandard\Config\ECSConfig;

return ECSConfig::configure()
    ->withPaths([
        __DIR__ . '/config',
        __DIR__ . '/examples',
        __DIR__ . '/resources',
        __DIR__ . '/src',
        __DIR__ . '/tests',
    ])

    // add a single rule
    ->withRules([
        NoUnusedImportsFixer::class,
    ])

    // add sets - group of rules, from easiest to more complex ones
    // uncomment one, apply one, commit, PR, merge and repeat
    ->withPreparedSets(
    //      spaces: true,
    //      namespaces: true,
    //      docblocks: true,
    //      arrays: true,
    //      comments: true,
    )
    ;
```

BUT, if we try to ask for help, 

```shell
vendor/bin/ecs --help
```

or just get current ECS version


```shell
vendor/bin/ecs --version
```

we got errors 

```text


 [ERROR] Pick at least one set in "->withPreparedSets()" in your ecs.php using named arguments, e.g.
         "->withPreparedSets(spaces: true)"


#0 /home/devilbox/data/php-compatinfo-db/ecs.php(24): Symplify\EasyCodingStandard\Configuration\ECSConfigBuilder->withPreparedSets()
#1 /home/devilbox/data/php-compatinfo-db/vendor-bin/ecs/vendor/symplify/easy-coding-standard/src/DependencyInjection/LazyContainerFactory.php(97): require('...')
#2 /home/devilbox/data/php-compatinfo-db/vendor-bin/ecs/vendor/symplify/easy-coding-standard/src/DependencyInjection/EasyCodingStandardContainerFactory.php(29): Symplify\EasyCodingStandard\DependencyInjection\LazyContainerFactory->create()
#3 /home/devilbox/data/php-compatinfo-db/vendor-bin/ecs/vendor/symplify/easy-coding-standard/bin/ecs.php(126): Symplify\EasyCodingStandard\DependencyInjection\EasyCodingStandardContainerFactory->createFromFromInput()
#4 /home/devilbox/data/php-compatinfo-db/vendor-bin/ecs/vendor/symplify/easy-coding-standard/bin/ecs(5): require('...')
#5 /home/devilbox/data/php-compatinfo-db/vendor/bin/ecs(119): include('...')
#6 {main}
```


I suggest to avoid raising error when help or version are asked !

--- a/templates/include/prepared_sets_php74.php.inc
+++ b/templates/include/prepared_sets_php74.php.inc
@@ -1,7 +1,7 @@
-    ->withSets([
+    //->withSets([
     //      \Symplify\EasyCodingStandard\ValueObject\Set\SetList::SPACES,
     //      \Symplify\EasyCodingStandard\ValueObject\Set\SetList::NAMESPACES,
     //      \Symplify\EasyCodingStandard\ValueObject\Set\SetList::DOCBLOCKS,
     //      \Symplify\EasyCodingStandard\ValueObject\Set\SetList::ARRAYS,
     //      \Symplify\EasyCodingStandard\ValueObject\Set\SetList::COMMENTS,
-    ])
+    //])

--- a/templates/include/prepared_sets_php80.php.inc
+++ b/templates/include/prepared_sets_php80.php.inc
@@ -1,7 +1,7 @@
-    ->withPreparedSets(
+    //->withPreparedSets(
     //      spaces: true,
     //      namespaces: true,
     //      docblocks: true,
     //      arrays: true,
     //      comments: true,
-    )
+    //)

